test/unit/compiler/passes/generate-bytecode.spec.js

```diff
@@ -361,11 +356,20 @@ describe("compiler pass |generateBytecode|", function() {
   });
 
   describe("for group", function() {
+    let grammar = "start = ('a')";
+
     it("generates correct bytecode", function() {
-      expect(pass).to.changeAST("start = ('a')", bytecodeDetails([
+      expect(pass).to.changeAST(grammar, bytecodeDetails([
         18, 0, 2, 2, 22, 0, 23, 1   // <expression>
       ]));
     });
+
+    it("defines correct constants", function() {
```

Comment:
This is added because I noticed, that for `group` node I forgot to update my PR for it, but tests did not tell me about that.